traitsui/qt4/list_str_editor.py

```diff
@@ -170,16 +170,16 @@ def init(self, parent):
 
         # Make sure we listen for 'items' changes as well as complete list
         # replacements:
-        self.context_object.on_trait_change(
-            self.update_editor, self.extended_name + "_items", dispatch="ui"
+        self.context_object.observe(
+            self.update_editor, self.extended_name + ":items", dispatch="ui"
```

Comment:
This change conflicts with the comment string - the comment says we should listen to changes in items as well as complete replacements - but it looks like we're not only listening to changes in items?

Reply:
Yeah I agree that is misleading.  Previously I believe the on_trait_change code was only listening to items changes so code should be consistent.  It is possible the code is supposed to listen to both as the comment says... It is hard to tell, and I'm unsure if we want to change behavior, even if the change is to the _right_ behavior.
Should I replace the `:` with a `.`?

Reply:
What happens if you switch `:` to `.`? Does it break the testsuite and/or examples? I'll try to git blame and see how this code evolved.

Reply:
No, I'm not seeing anything break

Reply:
I was hoping that the comment string was out of date because of some changes - but neither the comment string nor the code itself changed functionally since it was first checked in.

Let's go from `:` to `.` and see what breaks.